### ISSUE TITLE
Compiler crash when using special integers sizes

### ISSUE THREAD

**Issue by Inky-developer** [ziglang/zig#7322]
After reading the first lines of the zig documentation, I found out that zig supports special integers sizes like i7 or i65535. However, the following program:
```zig
const std = @import("std");
const print = std.debug.print;

pub fn main() void {
    var strange_int: i129 = 14;
    print("{}\n", .{strange_int});
}
```

fails with: 
```
>>> zig run hello.zig 
Assertion failed at /home/andy/dev/bootstrap-zig/zig/src/stage1/analyze.cpp:9586 in get_llvm_type. This is a bug in the Zig compiler.
Unable to dump stack trace: debug info stripped
```
Interestingely, when using an integer type of `i185` or higher, the error changes to:
```
LLVM ERROR: Unsupported library call operation!
```
integers below `i129` seem to work fine for me.

I am using windows with zig version 0.7.0.

**Comment by mikdusan** [comment]
with latest compiler:
- compiler no longer asserts
- appears this was fixed after 0.10.0
- it may have been partially fixed at 0.10.0 but missing symbol `__umodei4`

<details>
<summary>repro.zig</summary>

```zig
const std = @import("std");
const print = std.debug.print;

pub fn main() void {
    var num1: i129 = 14;
    print("0x{x}\n", .{num1});

    var num2: i185 = 15;
    print("0x{x}\n", .{num2});

    var num3: i185 = std.math.maxInt(i185);
    print("0x{x}\n", .{num3});

    var num4: i256 = @as(i256, num1) + @as(i256, num2) + @as(i256, num3);
    print("0x{x}\n", .{num4});
}
```
</details>

```sh
$ uname -a
Linux archlinux 6.1.2-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 31 Dec 2022 17:40:35 +0000 x86_64 GNU/Linux

$ zig version
0.11.0-dev.1247+87b223428

$ zig run repro.zig
0xe
0xf
0xffffffffffffffffffffffffffffffffffffffffffffff
0x1000000000000000000000000000000000000000000001c
```

**Comment by nektro** [comment]
this was a duplicate of https://github.com/ziglang/zig/issues/1485 and fixed with the switch to llvm 15
